docker-compose.test.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.5' 
+services: 
+  test:
+    environment:
+      REDIS_HOST: redis
+      REDIS_PASSWORD: redispassword
+      OAUTH_APP_ID: ${OAUTH_APP_ID}
+      DRONE_COMMIT_SHA: FOO
+      CC_TEST_REPORTER_ID: ${CC_TEST_REPORTER_ID}
+      CI: ${CI}
+      CIRCLECI: ${CIRCLECI}
+      CIRCLE_SHA1: ${CIRCLE_SHA1}
+      CIRCLE_BRANCH: ${CIRCLE_BRANCH}
+      GIT_COMMITTED_AT: ${GIT_COMMITED_AT}
+      OAUTH_APP_SECRET: ${OAUTH_APP_SECRET}
+      DEFAULT_URL_HOST: web
+      SELENIUM_URL: http://selenium:4444/wd/hub
+      APP_HOST: test
+      OAUTH_APP_URL: ${OAUTH_APP_URL}
+      POSTGRES_DB: scholarsphere
+      POSTGRES_USER: scholarsphere
+      POSTGRES_PASSWORD: scholarsphere
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID:-scholarsphere}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY:-scholarsphere}
+      AWS_BUCKET: scholarsphere
+      AWS_REGION: us-east-1
+      S3_ENDPOINT: http://minio:9000
+      RAILS_ENV: ${RAILS_ENV:-development}
```

Comment:
@whereismyjetpack I'm assuming this is the test build for running rspec. Wouldn't you want to default to "test" here?

Reply:
yup. good eye.